swagger/v1/openapi.json

```diff
@@ -2794,6 +2794,22 @@
               "default": ""
             }
           },
+
+          {
+            "name": "tags",
+            "in": "query",
+            "required": false,
+            "schema": {
+              "type": "array",
```

Comment:
Not entirely sure about this, I stole it [from here](https://github.com/RedHatInsights/patchman-engine/blob/ffcac0203ac7fc1129daf6eab04f7083ac4f083e/docs/openapi.json#L128) but I guess this is the best match :shrug:

Reply:
Yeah, it's kinda different from what the API actually returns within data.

Reply:
This is specified in the API docs

Reply:
Well, there are some specs about how is it queried and how is it stored and they are different...

Reply:
You did not add a usage example :disappointed: Within systems_spec.rb, we should show how to use the tags and an example response.

Reply:
:thinking: I'm not entirely sure what does this mean